network-api/networkapi/buyersguide/views.py

```diff
@@ -0,0 +1,16 @@
+from django.shortcuts import render
+from networkapi.buyersguide.models import Product
+from django.contrib.auth.decorators import login_required
+
+
+# Create your views here.
+@login_required
```

Comment:
is this "for now" or are these pages supposed to only be visible to ourselves?

Reply:
For now- basically a feature flag so we can merge this in without risk of it being stumbled upon.

Reply:
Can we put a comment in here with a refernce to a github issue along the lines of "the login requirement for buyersguide"?